docs/flow/modin/engines/dask/pandas_on_dask/frame/data.rst

```diff
@@ -0,0 +1,12 @@
+PandasOnDaskFrame
+"""""""""""""""""
+
+The class is specific implementation of ``BasePandasFrame`` class using Dask as an execution engine.
+It serves an intermediate level between ``pandas`` query compiler and ``PandasOnDaskFrameManager``.
```

Comment:
We should probably specify that it is the implementation of the dataframe algebra for Dask.

Reply:
Probably, yes. Specified.